tuichain/api/views/investments.py

```diff
@@ -59,31 +59,29 @@ def create_investment(request):
     loan = Loan.objects.filter(id=loan_id).first()
 
     if loan is None:
-        return Response(
-            {"error": "Unexistent Loan Request"}, status=HTTP_404_NOT_FOUND
-        )
+        return Response({"error": "Unexistent Loan"}, status=HTTP_404_NOT_FOUND)
 
     if loan.student == user.id:
         return Response(
-            {"error": "Cannot invest in your own Loan Request"},
+            {"error": "Cannot invest in your own Loan"},
             status=HTTP_403_FORBIDDEN,
         )
 
     if not loan.validated:
         return Response(
-            {"error": "The given Loan Request is not validated yet"},
+            {"error": "The given Loan is not validated yet"},
             status=HTTP_403_FORBIDDEN,
         )
 
     if not loan.active:
         return Response(
-            {"error": "The given Loan Request is not active anymore"},
+            {"error": "The given Loan is not active anymore"},
             status=HTTP_403_FORBIDDEN,
         )
 
     if loan.student == user:
         return Response(
-            {"error": "A user cannot invest in its own Loan Requests"},
+            {"error": "A user cannot invest in its own Loan"},
             status=HTTP_403_FORBIDDEN,
         )
 
```

Comment:
You can ignore this changes since those functions won't be in the next PR

Reply:
Done